ports/krabsetw/portfile.cmake

```
@@ -0,0 +1,13 @@
+vcpkg_fail_port_install(MESSAGE "krabsetw currently only supports Windows" ON_TARGET "UWP" "LINUX" "OSX" "FREEBSD" "ANDROID" "MINGW" "IOS")

```

```suggestion
vcpkg_fail_port_install(ON_TARGET "UWP" "LINUX" "OSX" "FREEBSD" "ANDROID" "MINGW")
```
`osx` is duplicated with `ios`.

---

Changed that